### ISSUE TITLE
PaymentMethod.create() doesn't accept all allowed params

### ISSUE THREAD

**Issue by jiujitsu** [braintree/braintree_python#45]
According to this: https://developers.braintreepayments.com/javascript+python/reference/request/payment-method/create

It looks like this method should allow these additional parameters:
* number
* expiration_date
* expiration_month
* expiration_year
* cardholder_name
* cvv
* venmo_sdk_payment_method_code

However, trying to use any combination of those results in an error such as:
```python
  vi +9   /usr/local/lib/python2.7/dist-packages/braintree/payment_method.py  # create
    return Configuration.gateway().payment_method.create(params)
  vi +22  /usr/local/lib/python2.7/dist-packages/braintree/payment_method_gateway.py  # create
    Resource.verify_keys(params, PaymentMethod.create_signature())
  vi +17  /usr/local/lib/python2.7/dist-packages/braintree/resource.py  # verify_keys
    raise KeyError("Invalid keys: " + keys_string)
KeyError: 'Invalid keys: expiration_month, cardholder_name, expiration_year, cvv, number'
```

**Comment by wjr1985** [comment]
Hi jiujitsu,

We've just released version 3.11.0 which will enable these values. After upgrading, if you're still running into trouble using these params, let us know.

Thanks,

Bill
Braintree Developer

**Comment by jiujitsu** [comment]
Awesome, works perfectly, thanks!
